crates/llm/src/lib.rs

```diff
@@ -1,8 +1,61 @@
-//! The `llm` crate provides a unified interface for loading and using
-//! Large Language Models (LLMs) such as LLaMA.
+//! This crate provides a unified interface for loading and using
+//! Large Language Models (LLMs). The following models are supported:
 //!
-//! At present, the only supported backend is GGML, but this is expected to
+//! - [BLOOM](https://huggingface.co/docs/transformers/model_doc/bloom) (inference produces garbled results)
+//! - [GPT-2](https://huggingface.co/docs/transformers/model_doc/gpt2)
+//! - [GPT-J](https://huggingface.co/docs/transformers/model_doc/gptj)
+//! - [LLaMA](https://huggingface.co/docs/transformers/model_doc/llama)
+//! - [GPT-NeoX](https://huggingface.co/docs/transformers/model_doc/gpt_neox)
```

Comment:
I'd suggest linking to the crates here (e.g. https://crates.io/crates/llm-neox), as you already link to HF in each crate's doc